src/server/saved_objects/service/saved_objects_client.d.ts

```diff
@@ -25,7 +25,7 @@ export interface BaseOptions {
 
 export interface CreateOptions extends BaseOptions {
   id?: string;
-  override?: boolean;
+  overwrite?: boolean;
```

Comment:
@legrego @weltenwort Git blame is giving me a hard time tracking down who originally wrote this but since you have been active in this file, do you mind confirming that this change to `overwrite` is actually correct? Thanks!

Reply:
I moved that code here from the spaces plugin. Looking at the `SavedObjectsClient` implementation, `overwrite` indeed looks correct.

Reply:
Thanks @weltenwort !